src/page-content/shared/content-team.tsx

```diff
@@ -255,16 +255,133 @@ export const team: TeamProps = {
+        }
+      ],
+      img: (
+        <StaticImage
+          placeholder="blurred"
+          layout="constrained"
+          aspectRatio={1}
+          loading="lazy"
+          className="responsive"
+          alt="Tobias Sahl"
+          src="../../images/team/tobias.png"
+        />
+      )
+    },
+    {
+      name: 'Michael Leyman',
+      position: 'Engineering',
       social: [
+        {
+          network: SocialNetworks.github,
+          href: 'https://github.com/MicLeey'
+        }
+      ],
+      img: (
+        <StaticImage
+          placeholder="blurred"
+          layout="constrained"
+          aspectRatio={1}
+          loading="lazy"
+          className="responsive"
+          alt="Michael Leey"
+          src="../../images/team/michael.png"
+        />
+      )
+    },
+    {
+      name: 'Rinor Gashi',
+      position: 'Marketing',
+      social: [
+        {
+          network: SocialNetworks.github,
+          href: 'https://github.com/RinorG'
+        },
+        {
+          network: SocialNetworks.linkedin,
+          href: 'https://www.linkedin.com/in/rinor-gashi-1aab781bb/'
+        }
+      ],
+      img: (
+        <StaticImage
+          placeholder="blurred"
+          layout="constrained"
+          aspectRatio={1}
+          loading="lazy"
+          className="responsive"
+          alt="Rinor Gashi"
+          src="../../images/team/rinor.png"
+        />
+      )
+    },
+    {
+      name: 'Anirudh Oppiliappan',
+      position: 'Engineering',
+      social: [
+        {
+          network: SocialNetworks.github,
+          href: 'https://github.com/icyphox'
+        },
         {
           network: SocialNetworks.web,
-          href: 'https://leeatchison.com/'
+          href: 'https://icyphox.sh'
+        }
+      ],
+      img: (
+        <StaticImage
+          placeholder="blurred"
+          layout="constrained"
+          aspectRatio={1}
+          loading="lazy"
+          className="responsive"
+          alt="Anirudh Oppiliappan"
+          src="../../images/team/anirudh.png"
+        />
+      )
+    },
+    {
+      name: 'Leonie Habermann',
+      position: 'Operations',
+      social: [
+        {
+          network: SocialNetworks.github,
+          href: 'https://github.com/haberml'
+        }
+      ],
+      img: (
+        <StaticImage
+          placeholder="blurred"
+          layout="constrained"
+          aspectRatio={1}
+          loading="lazy"
+          className="responsive"
+          alt="Leonie Habermann"
+          src="../../images/team/leonie.png"
+        />
+      )
+    },
+    {
+      name: 'Rebecca Gong',
+      position: 'Finance',
+      social: [
+        {
+          network: SocialNetworks.github,
+          href: 'https://github.com/RebeccaGong'
         },
         {
           network: SocialNetworks.linkedin,
-          href: 'https://www.linkedin.com/in/leeatchison'
+          href: 'http://linkedin.com/in/rebecca-qian-g-24b86b16'
```

Comment:
Remember to **always** use `https` links everywhere.

```suggestion
          href: 'https://linkedin.com/in/rebecca-qian-g-24b86b16'
```

Reply:
patrik, thank you 🎯